app/src/renderer/js/main.js

```diff
@@ -510,8 +512,9 @@ document.addEventListener('DOMContentLoaded', () => {
       if (!outputPath) {
         return;
       }
-
-      app.kap.editorWindow.send('toggle-format-buttons', {enabled: false});
+      // Initially an exception is triggered that editorWindow is undefined
+      // Using getter here should make it 'safer'
```

Comment:
I think that is because of a race issue. Can you try moving this https://github.com/wulkano/kap/blob/46c63f6d9a95e30fbab301ccbc99c6fb821ddbb0/app/src/main/index.js#L470 up to line 445 in the same file?